connectors/src/connectors/google_drive/temporal/activities.ts

```diff
@@ -852,15 +903,96 @@ async function deleteOneFile(connectorId: ModelId, driveFileId: string) {
   return;
 }
 
-// When exploring the Google Drive API, we need to keep track of the folders
-// we already explored. This is done by storing the folderId in a table.
-// This function clean-up the list of folders already visited on previous instances
-// of the full sync workflow.
-export async function cleanupDedupList(connectorId: ModelId, runId: string) {
-  await GoogleDriveBFSDedup.destroy({
+export async function cleanupSyncedFolders(
+  connectorId: ModelId,
+  runId: number
+) {
+  await GoogleDriveSyncedFolder.destroy({
     where: {
       connectorId: connectorId,
-      runId: { [Op.not]: runId },
+      runAt: { [Op.not]: runId },
```

Comment:
I think either:
- We use a real runId, eg the activity Id
- We use a date in which case it should be lastSeenTs and an inequality with the startTimeTs of the run